src/Aquifer.Common/Messages/Models/ResourceContentVersionSimilarityComparisonTypes.cs

```diff
@@ -0,0 +1,9 @@
+namespace Aquifer.Common.Messages.Models;
+
+public enum ResourceContentVersionSimilarityComparisonTypes
```

Comment:
`Type` is singular for other `enum`s like this.
```suggestion
public enum ResourceContentVersionSimilarityComparisonType
```